packages/artifact/src/internal/find/get-artifact.ts

```diff
@@ -49,7 +49,9 @@ export async function getArtifactPublic(
 
   if (getArtifactResp.data.artifacts.length === 0) {
     throw new ArtifactNotFoundError(
-      `Artifact not found for name: ${artifactName}`
+      `Artifact not found for name: ${artifactName}\n
```

Comment:
You don't need these extra newlines at the end, the multi-line string will already add them